tester_app/client/assets/stylesheets/_bootstrap-variables-customization.scss

```diff
@@ -0,0 +1,18 @@
+// This file is used as part of the bootstrap-sass-loader customization
```

Comment:
fixed, we've renamed this file to `_pre-bootstrap` and extracted the non-bootstrap code (all of it) to a new file named `react-on-rails-sass-helper`